src/main/jbake/content/blog/2016/12/16/hawkular-apm-vertx-opentracing-openshift.adoc

```diff
@@ -0,0 +1,60 @@
+= Hawkular APM improvements for OpenShift
+Juraci Paixão Kröhling
+2016-12-16
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, vertx, openshift
+
+On our last release, we added a few improvements for applications deployed in
+OpenShift, such as an automatic way of recognizing the service's name. This
```

Comment:
s/automatic way of recognizing the service's name/automatic way to recognize the name of the service/